ch_exporter/metrics.py

```diff
@@ -43,8 +44,9 @@ class CHMetric(BaseModel):
     factor: Optional[float] = None
     size: Optional[float] = None
     count: Optional[int] = None
-    prometheus_metric: None = None
 
+    _prometheus_metric: Any = PrivateAttr()
+    _active_label_values_by_node: dict[str, set[tuple[str]]] = PrivateAttr(default_factory=lambda: defaultdict(set))
```

Comment:
Note: Since the public prometheus API does not provide functionality to get or clear existing label values by a subset of values (node in our case), we maintain the active labels by node on our own.